mediator/fd_cesm.yaml

```diff
@@ -318,6 +318,10 @@
        canonical_units: 1e-6 mol/mol
        description: atmosphere export - prognostic CO2 at the lowest model level
      #
+     - standard_name: Sa_o3
+       canonical_units: 1e-9 mol/mol
```

Comment:
@fvitt can you please check these units? I took a guess that CAM's ozone is specified in ppbv, but I'm not sure if that's right.